src/sagas/activity/sagas.js

```diff
@@ -3,17 +3,14 @@ import { eventChannel } from 'redux-saga';
 
 import { fetchMessages, getEventEmitter, addDummyMessage, MESSAGING_QUEUE_JOB_ADDED } from './serviceFunctions';
```

Comment:
Do we really need to use service functions? Why don't we put everything inside the sagas?

Reply:
Yes, because sagas are internal application functions, while libs are external at the moment. It's always better to separate internal code from external libraries to not depend on them too much.